core/block_dragger.js

```diff
@@ -223,6 +218,12 @@ Blockly.BlockDragger.prototype.dragBlock = function(e, currentDragDeltaXY) {
     // Prevent unnecessary add/remove class calls.
     this.updateCursorDuringBlockDrag_();
   }
+
+  if (this.dragTarget_ !== oldDragTarget) {
```

Comment:
I think these were moved because they have to get called after `update` on the insertion marker manager so the `wouldDelete_` value is up to date? If that is the case, I would add a comment since this connection is non obvious and we don't want these getting moved in the incorrect order.

Reply:
Comment added.